tests/integration/fixtures/__init__.py

```diff
@@ -265,5 +280,8 @@ def request(self, method, path, **kwargs):
     def post(self, path, **kwargs):
         return self.request("post", path, **kwargs)
 
+    def req_options(self, path, **kwargs):
```

Comment:
```suggestion
    def options(self, path, **kwargs):
```

Reply:
As far as I remember I originally called it `options` but  the name clashes with an existing field name. I'll check and see how I can clean it up.